packages/contracts-bedrock/contracts/universal/OptimismMintableERC20.sol

```diff
@@ -16,12 +16,12 @@ contract OptimismMintableERC20 is IOptimismMintableERC20, ILegacyMintableERC20,
     /**
      * @notice Address of the corresponding version of this token on the remote chain.
      */
-    address public remoteToken;
+    address immutable public remoteToken;
 
     /**
      * @notice Address of the StandardBridge on this network.
      */
-    address public bridge;
+    address immutable public bridge;
```

Comment:
This should be SCREAMING_SNAKE a la #3834.

Reply:
I don't love making these screaming snake case to be honest, it feels a bit annoying to keep both interfaces

Reply:
This code hasn't run in production so I don't think we need to keep both interfaces. (Assuming that by "both interfaces" you mean [like this](https://github.com/ethereum-optimism/optimism/pull/3834/files#diff-952f0a0004b0405b194415ffab201f3d55ec3f0937edcab0ce08fd38dbbf0232R59-R60).

Reply:
Offchain services operating on the tokens may be running in production and we shouldn't break them

Reply:
You're right, so we need to maintain the interface with camelcase getters :)

But it's not clear to me why these immutables are a special case different from (for example) `OptimismMintableERC721` where we recently made the same change:

https://github.com/ethereum-optimism/optimism/blob/87702c741226ad4eb8a561dba349694e37c465a6/packages/contracts-bedrock/contracts/universal/OptimismMintableERC721.sol#L95-L104

Reply:
Ok, you are right, updating and pushing code that follows the screaming snake case pattern now

Reply:
TY!